parlai/crowdsourcing/tasks/turn_annotations_static/webapp/src/components/task_components.jsx

```diff
@@ -25,6 +26,18 @@ var showEnabledCssNextButton = function () {
   document.getElementById('submit-button').style = '';
 }
 
+var validateFreetextRepsonse = function (response) {
+  // require more than 10 characters, at least 1 vowel, and at least 2 words in the response.
+  var charCount = response.length;
+  var wordCount = response.split(' ').length;
+  var numVowels = response.match(/[aeiou]/gi);
+  if (charCount > 10 && wordCount > 1 && numVowels && numVowels.length > 1) {
+    return true;
+  } else {
+    return false;
+  }
+}
```

Comment:
is it possible to pass those checkers from backend to front-end? @JackUrb

Reply:
minor: it would be better to set thresholds of charCount, wordCount, and numVowels configurable.

Reply:
Also, you might want to talk to @mojtaba-komeili - I know he's also doing per-message checking on the frontend, so it'd be good to reuse code with him if possible

Reply:
In this case, as it's a static task there's no real way to communicate with the backend. The validator needs to be written in JS, but it could be configurable.